src/dodal/common/coordination.py

```diff
@@ -49,7 +49,7 @@ def scan(x: Movable = inject("stage_x"), start: float = 0.0 ...)
         from bluesky.utils import MsgGenerator
         from dodal.beamlines import i22
 
-        def my_plan(detector: Readable = i22.saxs(connect_immediately=False)) -> MsgGenerator:
+        def my_plan(detector: Readable = i22.saxs()) -> MsgGenerator:
```

Comment:
Must: Can you also update the docs and do a grep for `connect_immediately=False` in case we missed it anywhere else?

Reply:
Never mind, you've already done it, my bad, sorry!